synchronizer/connectors/jira.py

```diff
@@ -62,6 +64,35 @@ def export_worklogs(self, worklogs):
                 print(err)
                 raise self.ExportException(i)
 
+    def validate_worklogs(self, worklogs):

```

Comment:
Let's pass only issue_id. I think that this is enough for this method to do validation for now.

Reply:
Even better. Let's replace this method with `validate_issue(issue_id)` (which return True or False, do not forget to write it in method description). In this case you should also add a method `Synchronization.validate_worklogs` which will call `validate_issue` in the loop for all worklogs.

This solution will keep code clean and extendable for other types of validations.

Reply:
Also note that in new realization (from previous comment) we will have connectors without "tight" relations with models on TimeSync. This is important.